src/components/approval-type/ApprovalType.vue

```diff
@@ -8,87 +11,81 @@
       >
         <label :class="{ 'error-text': invalidSection }">Approval Type</label>
       </v-col>
+
       <v-col
         cols="12"
         sm="9"
-        class="mt-n4"
+        class="mb-n5"
       >
         <v-radio-group
-          v-model="approvalTypeSelected"
```

Comment:
It seemed redundant to use both a v-model property and handle the change event so I went with `:value` and `@change` to keep things consistent with the other components in here.